src/AuthLDAP.php

```diff
@@ -3860,6 +3855,10 @@ public static function manageValuesInSession($options = [], $delete = false)
                     }
                 }
 
+                if ($entity->getField('entity_ldapfilter') != NOT_AVAILABLE) {
+                    $_SESSION['ldap_import']['entity_filter'] = $entity->getField('entity_ldapfilter');
+                }
```

Comment:
Is this possible to have an empty value in `$_SESSION['ldap_import']['entities_id']` and therefore to have no entity loaded? In this case, the `$entity->getField('entity_ldapfilter')` would be executed on an unloaded `$entity`, and I guess it would fail.

Reply:
I don't think so, but it's mainly the second condition that prevents us from entering the IF `($entity->getField(‘authldaps_id’) > 0)`, because “Default server” corresponds to the empty value.

It's `getField()` that returns `NOT_AVAILABLE` when it can't find the value.

Reply:
Should be OK:
```
$entity = new Entity();
$entity->getField('entity_ldapfilter'); // ==> "N/A" 
```